app/src/test/java/com/github/se/cyrcle/model/user/UserTest.kt

```diff
@@ -5,17 +5,6 @@ import org.junit.Assert.assertNotEquals
 import org.junit.Test
 
 class UserTest {
-  @Test
-  fun testConstructor() {
```

Comment:
Why did you removed this test ? Seems relevant to me.

Reply:
Testing the default constructor of a class sounds really useless. That is up to the dev Team of Kotlin.